README.md

```diff
@@ -308,6 +308,71 @@ Just like with the `docker compose` CLI, you can use the global `--file | -f` co
 
 In addition to the project compose files, an optional Preevy-specific Compose file can be used. Preevy attempts to load files named `compose.preevy.yaml`, `compose.preevy.yml`, `docker-compose.preevy.yaml` or `docker-compose.preevy.yml`. If one of these exists, it is loaded BEFORE the project composes file(s). The name of the Preevy-specific compose file can be overridden by specifying the argument `--system-compose-file`.
 
+### Environment Variables
```

Comment:
Try to minimize it and a link to recpies example you've created (and docker compose relevant documentation)